components/blocks/About.vue

```diff
@@ -114,8 +110,16 @@ export default {
   computed: {
     tocData() {
       const ogData = this.data;
-      const sortOrder = ['mission','people','opportunities','facilities','diversity'];
-      const sortedData = ogData.sort(function(a, b){return sortOrder.indexOf(a.slug) - sortOrder.indexOf(b.slug)});
+      const sortOrder = [
+        'mission',
+        'people',
+        'opportunities',
+        'facilities',
+        'diversity',
+      ];
+      const sortedData = ogData.sort(function (a, b) {
```

Comment:
```suggestion
      const sortedData = ogData.sort((a, b) => {
```
just to be consistent